src/rpm/builder.rs

```diff
@@ -389,7 +387,6 @@ impl RPMBuilder {
         let mut header = Vec::with_capacity(128);
         header_idx_tag.write(&mut header)?;
 
-        #[cfg(feature = "signature-meta")]
```

Comment:
Yes, we should always build this, so removing the signature-meta tag here is correct.